harvester_e2e_tests/integrations/test_upgrade.py

```diff
@@ -494,6 +499,10 @@ def test_degraded_volume(
         3. Immediately upgrade Harvester.
         4. Upgrade should fail.
         """
+        # https://github.com/harvester/harvester/issues/6425
+        if cluster_state.size < 3:
+            pytest.skip(f"Degraded only checked when nodes >= 3, skip for {cluster_state.size}.")
```

Comment:
Typo: Degraded volumes only checked when nodes >= 3 :)

Reply:
Fixed in 702026f.
![image](https://github.com/user-attachments/assets/98f20343-960a-4440-a55f-de615313f28e)